src/@ionic-native/plugins/preview-any-file/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { Plugin, Cordova, IonicNativePlugin } from '@ionic-native/core';
+/**
+ * @name Preview Any File
+ * @description
+ * This plugin does something
+ *
+ * @usage
+ * ```typescript
+ * import { PreviewAnyFile } from '@ionic-native/preview-any-file';
+ *
+ *
+ * constructor(private previewAnyFile: PreviewAnyFile) { }
+ *
+ * ...
+ *
+ *
+ * this.previewAnyFile.preview('file://filepath.ext')
+ *   .then((res: any) => console.log(res))
+ *   .catch((error: any) => console.error(error));
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'PreviewAnyFile',
+  plugin: 'cordova-plugin-preview-any-file', // npm package name, example: cordova-plugin-camera
+  pluginRef: 'PreviewAnyFile', // the variable reference to call the plugin, example: navigator.geolocation
+  repo: 'https://github.com/mostafa-mansour1/previewAnyFile', // the github repository URL for the plugin
+  install: '', // OPTIONAL install command, in case the plugin requires variables
+  installVariables: [], // OPTIONAL the plugin requires variables
+  platforms: ['Android', 'iOS'] // Array of platforms supported, example: ['Android', 'iOS']
+})
+@Injectable()
+export class PreviewAnyFile extends IonicNativePlugin {
+  /**
+   * This function does something
```

Comment:
Please add a description

Reply:
please check the new commit 
```suggestion
   *    * this function return SUCCESS in success callback if the file successfully opened, if the content is base64 you have to write it into file by cordova-plugin-file

```